httpx/_client.py

```diff
@@ -1097,20 +1095,18 @@ def close(self) -> None:
         """
         Close transport and proxies.
         """
-        if not self.is_closed:
```

Comment:
So we're safe to assume that `.close()` can be called on transports multiple times without problem? I guess anything else would be a badly written transport..

Reply:
I think that's reasonable yeah. Also, we're in line with requests here.

I guess we *could* tweak things in a future version bump so that calling `.close()` on an already closed client would result in a loud explicit error. (Just as we'd also like to tighten things up so that calling `.request()` on a closed client should result in an error.)